goodtables/spec.json

```diff
@@ -135,13 +135,13 @@
           "message": "There is a missing header in column {column_number}",
           "description": "Based on the schema there should be a header that is missing in the first row of the data source.\n\n How it could be resolved:\n - Add the missing column to the data source or remove the extra field from the schema\n - If this error should be ignored disable `missing-header` check in {validator}."
         },
-        "non-castable-value": {
-          "name": "Non-Castable Value",
+        "type-or-format-error": {
+          "name": "Type or Format Error",
           "type": "schema",
           "context": "body",
           "weight": 9,
-          "message": "Row {row_number} has non castable value {value} in column {column_number} (type: {field_type}, format: {field_format})",
-          "description": "The value can't be cast based on the schema type and format for this field.\n\n How it could be resolved:\n - If this value is not correct, update the value.\n - If this error should be ignored disable `non-castable-value` check in {validator}. In this case all schema checks for row values will be ignored."
+          "message": "The value {value} in row {row_number} and column {column_number} is not type {field_type} and format {field_format}",
```

Comment:
The error type is 'type __or__ format', but the messages are 'type __and__ format'. Will that always be correct?

Reply:
It's been reviewed on data quality spec level so I suppose it's correct. It's harder for non-native speaker to say but my understanding that:
- value must be this type and format BUT
- value has wrong type or format

Reply:
Anyway we just use spec from the upstream so it will be addressed on the spec level if needed.